tests/test_device.py

```diff
@@ -339,7 +339,7 @@ def test_check_validity_with_not_supported_operation_inverse(
 
         with pytest.raises(
             DeviceError,
-            match="Gate {} not supported on device {}".format("CNOT", "MockDevice"),
```

Comment:
I wonder why it got that way back in the day? 🤔

Reply:
Copy and paste from the main code, and replace the variables with fixed values :)